src/main/java/core/basesyntax/classes/figures/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.classes.figures;
+
+public class Circle extends Figure {
+    private final int radius;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.